src/reduce.js

```diff
@@ -27,13 +29,32 @@ var _reduce = require('./internal/_reduce');
  * @param {*} acc The accumulator value.
  * @param {Array} list The list to iterate over.
  * @return {*} The final, accumulated value.
- * @see R.reduced, R.addIndex
+ * @see R.reduced, R.addIndex R.reduceRight
  * @example
  *
- *      var numbers = [1, 2, 3];
- *      var plus = (a, b) => a + b;
+ *      R.reduce(R.subtract, 0, [1, 2, 3, 4]) // => ((((0 - 1) - 2) - 3) - 4) = -10
+ *                -               -10
+ *               / \              / \
+ *              -   4           -4   4
+ *             / \              / \
+ *            -   3   ==>     -3   3
+ *           / \              / \
+ *          -   2           -1   2
+ *         / \              / \
+ *        0   1            0   1
+ *
+ *      R.reduceRight(R.subtract, 0, [1, 2, 3, 4]) // => (1 - (2 - (3 - (4 - 0)))) = -2
+ *          -               -2
+ *         / \              / \
+ *        1   -            1   3
+ *           / \              / \
+ *          2   -     ==>    2  -1
+ *             / \              / \
+ *            3   -            3   4
+ *               / \              / \
+ *              4   0            4   0
  *
```

Comment:
I don't think that the functions need to have both diagrams.  People interested in the difference can look back and forth between the `reduce` and `reduceRight` docs for this.